.github/workflows/test-unit.yml

```diff
@@ -28,19 +29,34 @@ jobs:
 
       - name: Checkout code
         uses: actions/checkout@v4
-      
+
       - name: Set TEST_KBC_PROJECTS
         uses: ./.github/actions/export-kbc-projects
         with:
           secrets: ${{ toJSON(secrets) }}
           projects: ${{ vars.TEST_KBC_PROJECTS }}
 
-      - name: Load Go cache
+      - name: Load Go build cache
         uses: ./.github/actions/go-cache
         with:
-          key: unit-${{ matrix.name }}
+          key: unit
+
+      # Fix timestamps
+      - name: restore timestamps
+        uses: chetan/git-restore-mtime-action@v2
+
+      #- name: Change modtime of files to 1 unix timestamp
+      #  shell: bash
+      #  run: |
+      #    pwd
+      #    for i in ~/go/pkg; do
+      #      find "$i" -type f -exec touch -r {} -d 'Thu Jan 01 1970 00:00:01' {} \; || true
+      #    done
```

Comment:
We still do not know whether fixer of timestamp is needed. Once I tried the CI and it worked also without restoring timestamps.